### DISCUSSION TITLE
LibreOffice HEIC conversion

### DISCUSSION THREAD

**Discussion by ytilotti** [gotenberg/gotenberg#549] [Q&A] [answered]
Hi all,

Do you know if Heic files will be convertible on libreoffice (and Gotenberg)?
Why do not add a convertor in Gotenberg?

**Comment by VladaMamutova** [comment, marked as answer]
Hello!
I am also interested in this question. The file format is very popular nowadays. It looks like this limitation in gotenberg can be easily removed, because the `loffice` utility (LibreOffice 24.2.5.2), which is used by gotenberg-8.10.0, converts the heic-file to pdf correctly.
![heic-convertation](https://github.com/user-attachments/assets/dedd283c-e9f0-4fc9-add0-8b8152e3ad59)
